sensor_msgs/PointCloud2 nube_objetos
coordObjetos[] coord_objetos
float64 yaw
